backend/controller/observability/calls.go

```diff
@@ -0,0 +1,72 @@
+package observability
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/alecthomas/types/optional"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric"
+	"go.opentelemetry.io/otel/metric/noop"
+
+	schemapb "github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1/schema"
+	"github.com/TBD54566975/ftl/backend/schema"
+	"github.com/TBD54566975/ftl/internal/observability"
+)
+
+const (
+	callMeterName         = "ftl.call"
+	callVerbRefAttr       = "ftl.call.verb.ref"
+	callFailureModeAttr   = "ftl.call.failure_mode"
+	callRunTimeBucketAttr = "ftl.call.run_time_ms.bucket"
```

Comment:
nit: would it make sense to log the run time attribute as well as its bucket?

Reply:
I wish... the non-bucketized value is basically useless because we can only use these for filtering, and they're nearly all unique. Then, datadog ends up showing nearly every single value in its filter dropdown, which fills it up to its max size and you can't see explore the values of any other attributes. I ended up removing that from async calls too because it made the UI so frustrating to use :/